### PR TITLE
MTM-57760: Change log for Map ID Token Parameters in SSO configuration

### PR DESCRIPTION
**Pull request by kowa-c8y**

### CODE REVIEW

File: content/change-logs/platform-services/cumulocity-1020-589-0-support-for-id-token-procured-by-an-external-iam-server-in-the-cumulocity-sso-authentication-process.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-57760
+version: 1020.589.0
+---
+Support for ID tokens provided by external IAM servers has been added and can now be used in the authentication process via SSO. Previously, only access tokens were supported by the Cumulocity platform. Now, the administrator can decide through the configuration whether user data retrieved from the external server, such as first name, last name, phone number, email, and roles, should be loaded from the access token or the ID token.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-1020-589-0-support-for-id-token-procured-by-an-external-iam-server-in-the-cumulocity-sso-authentication-process.md]:
```suggestion
Support for ID tokens provided by external IAM servers has been added and can now be used in the authentication process via single sign-on (SSO). Previously, only access tokens were supported by the {{< product-c8y-iot >}} platform. Now, the administrator can decide through the configuration whether user data retrieved from the external server, such as first name, last name, phone number, email, and roles, should be loaded from the access token or the ID token.
```

File: content/change-logs/platform-services/cumulocity-1020-589-0-support-for-id-token-procured-by-an-external-iam-server-in-the-cumulocity-sso-authentication-process.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Support for ID token procured by an external IAM server in the Cumulocity SSO authentication process
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-1020-589-0-support-for-id-token-procured-by-an-external-iam-server-in-the-cumulocity-sso-authentication-process.md]:
```suggestion
title: Support for ID tokens provided by external IAM servers in the SSO authentication process
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@kgol-c8y @saran-govindarajan @pink-mongoose please approve

File: content/change-logs/platform-services/cumulocity-1020-589-0-support-for-id-token-procured-by-an-external-iam-server-in-the-cumulocity-sso-authentication-process.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  - value: change-QHu1GdukP
+    label: Feature
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-57760
+version: 1020.589.0
```
Comment (kgol-c8y) [inline content/change-logs/platform-services/cumulocity-1020-589-0-support-for-id-token-procured-by-an-external-iam-server-in-the-cumulocity-sso-authentication-process.md:15]: the fixed version in the ticket is different than here 
image
can you please cross check from which version its available?

Comment (kowa-c8y) [reply content/change-logs/platform-services/cumulocity-1020-589-0-support-for-id-token-procured-by-an-external-iam-server-in-the-cumulocity-sso-authentication-process.md:15]: This is the version of UI changes. Core changes are in 1020.589.0 version

Comment (kowa-c8y) [reply content/change-logs/platform-services/cumulocity-1020-589-0-support-for-id-token-procured-by-an-external-iam-server-in-the-cumulocity-sso-authentication-process.md:15]: Should we post here both versions or cumulocity-core is enough?

Comment (kgol-c8y) [reply content/change-logs/platform-services/cumulocity-1020-589-0-support-for-id-token-procured-by-an-external-iam-server-in-the-cumulocity-sso-authentication-process.md:15]: one is enough.

**Review by kgol-c8y — APPROVED** [review]

**Pull request merged**